docs/hive-integration/crds/agentClusterInstall-dualstack.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: test-agent-cluster-install
+  namespace: spoke-cluster
+spec:
+  apiVIP: 1.2.3.8
```

Comment:
I'm assuming this has to be IP4 due to other discussions I've heard but would be good to mention that somewhere.

Reply:
Correct. For dual-stack API and Ingress IPs have to be IPv4 (until Metal supports an array of addresses here; luckily there is already a Jira for this). As above, I'm happy to craft the doc explaining all this as soon as we have _some_ idea where it should live

Reply:
+1 for the doc!

Let's write this doc, and then add a comment with a link to the networking docs somewhere in this yaml.